src/components/Login/Login.js

```diff
@@ -1,16 +1,111 @@
 import React from 'react';
+import { Link } from 'react-router-dom';
+import { injectIntl } from 'react-intl';
+import { timeout } from '../../utils';
+import { Formik, Form } from 'formik';
+import { FieldGroup, InputFieldItem, PasswordFieldItem } from '../form-helpers/form-helpers';
 
-export default function({ location }) {
-  return (
-    <div>
-      {/* TODO: implement login */}
-      LOGIN. After login, redirect to{' '}
-      {location.state && location.state.from
+const fieldNames = {
+  user: 'user',
+  password: 'password',
+};
+
+/** Prepare empty values for all fields
+ * It is required because in another way, the fields are not marked as touched.
+ */
+const getFormInitialValues = () =>
+  Object.keys(fieldNames).reduce(
+    (accumulator, currentValue) => ({ ...accumulator, [currentValue]: '' }),
+    {},
+  );
+
+export default injectIntl(function({ intl, location }) {
+  const _ = (id, values) => intl.formatMessage({ id: id }, values);
+
+  const validate = (values) => {
+    const errors = {};
+
+    if (!values[fieldNames.user]) {
+      errors[fieldNames.user] = _('validation_messages.error_required_field');
+    } else if (!/^[A-Z0-9._%+-]+@[A-Z0-9.-]+\.[A-Z]{2,4}$/i.test(values[fieldNames.user])) {
+      // TODO: review if this is our desired validation
+      errors[fieldNames.user] = _('validation_messages.error_invalid_email_address');
+    }
+
+    if (!values[fieldNames.password]) {
+      // TODO: I think that password validation has a different format
+      errors[fieldNames.password] = _('validation_messages.error_required_field');
+    } else {
+      // TODO: validate password
+    }
+
+    return errors;
+  };
+
+  const onSubmit = async (values, { setSubmitting }) => {
+    // TODO: implement login submit
+    await timeout(1500);
+    setSubmitting(false);
+
+    window.location.href =
```

Comment:
Mmm, I think that we should use a different way of navigation (not totally sure, but almost).

Since we do not have backend ready yet, what do you think about removing this redirection?

Reply:
i don't think this is a problem because redirect to home page, what problem do you see?

Reply:
Creo que los redirect se hacen de otra manera, por eso te proponía no incluir ese código ya que igual no tenemos el login.

Pero no me parece mal que lo hagas completo, en la línea 108 tenés el redirect que tenemos que implementar: https://reacttraining.com/react-router/web/example/auth-workflow

Reply:
Listo agregado!